config/routes.rb

```diff
@@ -1,11 +1,18 @@
 Rails.application.routes.draw do
   root 'organizations#index'
-  resources :organizations, only: %i[new create]
+  resources :organizations, only: %i[new create index] do 
+    collection do
+      get 'login'
+      post 'process_login'
+    end
```

Comment:
In Organization controller there is only index action, where are other routes being used?